### PR TITLE
Added allow anonymous dropdown select in edit instances

### PR DESCRIPTION
**Pull request by singhsanket143**
The pull request is implementation for the feature listed in issue #16613. The allow anonymous functionality is now added to the edit instance feature. User can now anytime change the allow anonymous reply option by navigating to the edit advanced features section of an instance.

image

### CODE REVIEW

File: client/views/edit/edit.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ Template.edit.events({
     const stale = template.$('select[name=stale_length]').val();
     const isHidden = (template.$('select[name=visibility]').val() === 'hidden');
     const social = (template.$('select[name=social]').val() === 'on');
+    const anonymous = template.$("select[name=anonymous]").val() === "on"; // Fetch the anonymous value from the dropdown
```
Comment (peter-hank) [inline client/views/edit/edit.js]: Single quotes please :smiling_imp:

File: client/views/edit/edit.html [hunk 2]
```diff
@@ -10,7 +10,7 @@ <h1 class="formtitle">Edit</h1>
 		<div class="inputcontainer" style="text-align: center;">
```
Comment (peter-hank) [inline client/views/edit/edit.html:10]: Let's fix the formatting here.

P.S. You can do it in every html template, just remember to do it in a separate commit (and call it something like `Fixed formatting`).

**Review by peter-hank — CHANGES_REQUESTED** [review]

File: client/views/edit/edit.html (hunk 2 above)
Comment (singhsanket143) [reply client/views/edit/edit.html:10]: Updated the formatting

File: client/views/edit/edit.js (hunk 1 above)
Comment (singhsanket143) [reply client/views/edit/edit.js]: Updated the quotes here.

File: client/views/edit/edit.html (hunk 2 above)
Comment (singhsanket143) [reply client/views/edit/edit.html:10]: Updated the formatting

Comment (peter-hank) [inline client/views/edit/edit.html:10]: @singhsanket143 actually, can you fix the formatting in the whole file?

Comment (singhsanket143) [reply client/views/edit/edit.html:10]: @peter-hank Updated the file formatting

Comment (peter-hank) [reply client/views/edit/edit.html:10]: @singhsanket143 it's still not too great, can you use something like https://codebeautify.org/htmlviewer/ to format it? Thanks.

**Review by peter-hank — CHANGES_REQUESTED** [review]

File: client/views/edit/edit.html (hunk 2 above)
Comment (singhsanket143) [reply client/views/edit/edit.html:10]: @peter-hank  I used the following tool and have refactored the code. Please review it. Also, shall I use the same tool for refactoring in other branches also?

**Comment by peter-hank** [comment]
@singhsanket143 looks great, thanks

File: client/views/edit/edit.js [hunk 3]
```diff
@@ -32,6 +32,7 @@ Template.edit.events({
     const stale = template.$('select[name=stale_length]').val();
     const isHidden = (template.$('select[name=visibility]').val() === 'hidden');
     const social = (template.$('select[name=social]').val() === 'on');
+    const anonymous = template.$('select[name=anonymous]').val() === 'on'; // Fetch the anonymous value from the dropdown
```
Comment (peter-hank) [inline client/views/edit/edit.js]: @singhsanket143 can you move this comment up?

Comment (singhsanket143) [reply client/views/edit/edit.js]: @peter-hank Updated the comments here.
